### PR TITLE
Store text files in UTF-8

### PR DESCRIPTION
**Pull request by orgads**
Convert them on checkout.

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -1 +1,4 @@
 *.san filter=lfs diff=lfs merge=lfs -text
+strings.txt working-tree-encoding=cp1255
+/comi/RESOURCE/LANGUAGE.TAB working-tree-encoding=cp1255
+/dig/VIDEO/DIGTXT.TRS working-tree-encoding=cp1255
```
Comment (orgads) [inline .gitattributes]: Any other files I might have missed?

**Comment by BLooperZ** [comment]
Thanks,
does it means that text files will be shown as utf-8 on github, but when have the files locally they will be available as windows-1255?

**Comment by orgads** [comment]
Exactly.

File: .gitattributes (hunk 1 above)
Comment (BLooperZ) [reply .gitattributes]: FT has many TRS files, might be better to make it glob pattern `*.TRS`
LANGUAGE.TAB is also used for Grim Fandango if we decide to put it here as well,
and apparently the DIG can LANGUAGE.TAB as well and I would like to check it.
so `LANGUAGE.TAB` also good entry as is

Comment (orgads) [reply .gitattributes]: Done

**Pull request merged**